Makefile

```
@@ -83,6 +83,21 @@ kubeval: _site
 	! grep -C3 -P "invalid|\t\*" filtered.out
 	rm filtered.out
 
+PACKAGE_NAME?=github.com/projectcalico/calico
+helm-tests: vendor bin/helm helm-tests/fv.test versions-values.yaml
```

```suggestion
helm-tests: vendor bin/helm versions-values.yaml
```
There's no more `fv.test`, right?

---

good catch

---

Is it worth removing the `helm-tests/fv.test` entry in the [`.gitignore`](https://github.com/projectcalico/calico/pull/2504/files#diff-a084b794bc0759e7a6b77810e01874f2R15) file.